panic_handler/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+#![no_std]
+
+use libtock2::{
+    low_level_debug::{AlertCode, LowLevelDebug},
+    platform::Syscalls,
+    runtime::TockSyscalls,
+};
+
+#[panic_handler]
+unsafe fn panic_handler(_info: &core::panic::PanicInfo) -> ! {
+    report_panic()
+}
```

Comment:
Why did you put `panic_handler`'s functionality in a separate function?

Reply:
Was copied from `core/src/lang_items.rs`, fixed.